### DISCUSSION TITLE
Inconsistency of possible build states?

### DISCUSSION THREAD

**Discussion by cyqsimon** [fedora-copr/copr#2950] [General]
I was doing some CI scripting (see [here](https://github.com/cyqsimon/el-rust-pkgs-spec/blob/master/.github/workflows/rebuild-changed.yaml)) and therefore was searching through COPR source code to find out what the possible build states are.

I ran into those two different definitions of build states. In particular, `imported` and `unknown` differ.

https://github.com/fedora-copr/copr/blob/c381e3434ee617dbab4effcf347ddbe06b33bb98/common/copr_common/enums.py#L85-L98

https://github.com/fedora-copr/copr/blob/c381e3434ee617dbab4effcf347ddbe06b33bb98/frontend/coprs_frontend/coprs/filters.py#L206-L219

I would like to inquire which one I should reference, and also whether this is something that should be updated. Thanks.

**Comment by praiskup** [comment]
Thank you for the question. The Enum defined in copr common is the "definition" that other components "respect" -> filters is just "for the web-UI" description (we should move these two things onto one place one day).
